cli/command_snapshot_create.go

```diff
@@ -129,6 +133,32 @@ func snapshotSingleSource(ctx context.Context, rep *repo.Repository, u *snapshot
 
 	manifest.Description = *snapshotCreateDescription
 
+	if *snapshotCreateStartTime != "" {
```

Comment:
please move parsing to be done sooner. otherwise folks may spend hours uploading only to find out they had a typo in the flag and the snapshot will fail.

Reply:
Thanks for picking up on this - I didn't even think about that!